results/Analyzing_4_Million_Jupyter_Notebooks.md

```diff
@@ -15,7 +15,7 @@ I first queried all public, non-forked Jupyter notebooks, then downloaded them a
 
 ## *Owners*
 
-In July of 2017, there were 100,503 GitHub users with publicly shared Jupyter notebooks, at the time approximately 0.4% of all owners. Further, Adam revealed that 24.5% of these owners only hosted one notebooks on GitHub. Now, there are 414,614 owners, 1.16% of the nearly 36 million users ([GitHub search for users](https://github.com/search?utf8=%E2%9C%93&q=repos%3A%3E=0&type=Users&ref=advsearch&l=&l=)). Not far from the 2017 study, 28.7% of Jupyter users host only one notebook on GitHub.
+In July of 2017, there were 100,503 GitHub users with publicly shared Jupyter notebooks, at the time approximately 0.4% of all owners. Further, Adam revealed that 24.5% of these owners only hosted one notebooks on GitHub. Now, there are 414,614 owners, 1.16% of the nearly 36 million users ([GitHub search for users](https://github.com/search?utf8=%E2%9C%93&q=repos%3A%3E=0&type=Users&ref=advsearch&l=&l=)). Not far from the 2017 study, 28.7% of notebook owners host only one notebook on GitHub.
 
 Notebooks owners can either be individual users or organizations. Adam’s dissertation doesn’t mention the differences between owners and organizations, so there is no 2017 baseline to refer to. 4.72% of owners are organizations and they own 7.86% of all notebooks. The median number of notebooks per owner is 4 for both organizations and owners, but the mean number for organizations is much higher at 19.03 compared to 10.88 notebooks for users. Both distributions are skewed right, but more so for organizations. The most notebooks held by a single owner is 26,032, a record held by the organization [learn-co-students](https://github.com/learn-co-students).
```

Comment:
My correction was on "Notebooks owners can either be individual users or organizations"

Reply:
My bad, thank you!